### PR TITLE
Outlook v2 source connector

### PR DESCRIPTION
**Pull request by Paul-Cornell**
See:

- API: https://unstructured-53-outlook-source-2024-10-17.mintlify.app/api-reference/ingest/source-connectors/outlook
- Open source: https://unstructured-53-outlook-source-2024-10-17.mintlify.app/open-source/ingest/source-connectors/outlook

### CODE REVIEW

File: snippets/general-shared-text/outlook.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+The Outlook prerequisites:
+
+- The Outlook user's email address.
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook.mdx:3]:
```suggestion
The Outlook prerequisites:

<iframe
width="560"
height="315"
src="https://www.youtube.com/embed/9yESRp9pzv0"
title="YouTube video player"
frameborder="0"
allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
allowfullscreen
></iframe>

- The Outlook user's email address.
```

File: snippets/source_connectors/outlook.v2.py.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            access_config=OutlookAccessConfig(client_cred=os.getenv("OUTLOOK_APP_CLIENT_SECRET")),
+            client_id=os.getenv("OUTLOOK_APP_CLIENT_ID"),
+            tenant=os.getenv("OUTLOOK_APP_TENANT")
+        ),
+        partitioner_config=PartitionerConfig(
+            partition_by_api=True,
+            api_key=os.getenv("UNSTRUCTURED_API_KEY"),
+            partition_endpoint=os.getenv("UNSTRUCTURED_API_URL")
+        ),
+        embedder_config=EmbedderConfig(embedding_provider="huggingface"),
```
Comment (ninalopatina) [inline snippets/source_connectors/outlook.v2.py.mdx:39]: is there a reason these are all missing the chunking step before the embedding step?

**Review by ninalopatina — CHANGES_REQUESTED** [review]

File: snippets/source_connectors/outlook.v2.py.mdx (hunk 2 above)
Comment (Paul-Cornell) [reply snippets/source_connectors/outlook.v2.py.mdx:39]: Hmm...not sure. I'll add them now though!

Comment (Paul-Cornell) [reply snippets/source_connectors/outlook.v2.py.mdx:39]: What kind of chunking strategy here--is "by_title" reasonable?

Comment (Paul-Cornell) [reply snippets/source_connectors/outlook.v2.py.mdx:39]: Added "by_title".

**Review by ninalopatina — APPROVED** [review]

**Pull request merged**